app/views/layouts/_footer.html.erb

```diff
@@ -2,14 +2,16 @@
 	<hr class="hr_gradient">
 	<div class="row">
 		<div class="col-sm-12 text-center">
-			<small>Please send feedback or ideas for this case management app to <a href="mailto:DCAF.CMapp@gmail.com">DCAF.CMapp@gmail.com</a></small>
+			<small><%= t 'navigation.footer.please_send_feedback' %> <%= link_to 'DCAF.CMapp@gmail.com', mailto: 'DCAF.CMapp@gmail.com' %>.</small>
```

Comment:
tried really hard to stick the link in the il8n key and couldn't get it to work in 5m of poking. Not ideal but I'm not stressing about it.